### DISCUSSION TITLE
scripts/generate_baseline.py -k

### DISCUSSION THREAD

**Discussion by erefneb** [usnistgov/macos_security#51] [Q&A] [answered]
I am getting an error when trying to generate guidance from a generated baseline from build/baselines/$generated_baseline.yaml
As opposed to using baselines/$existing_baseline.yaml

scripts/generate_baseline.py -l
800-171
800-53r4_high
800-53r4_low
800-53r4_moderate
STIG
cnssi-1253
inherent
manual
n_a
permanent
supplemental

scripts/generate_baseline.py -k 800-53r4_high

scripts/generate_guidance.py build/baselines/800-53r4_high.yaml
Profile YAML: build/baselines/800-53r4_high.yaml
Output path: /Users/eric/Projects/macos_security_2021-02-09_17-24/build/800-53r4_high/800-53r4_high.adoc
Traceback (most recent call last):
  File "scripts/generate_guidance.py", line 1378, in <module>
    main()
  File "scripts/generate_guidance.py", line 1200, in main
    with open(rule_path[0]) as r:
IndexError: list index out of range

**Comment by brodjieski** [comment, marked as answer]
Which branch are you attempting to use?  It looks like 'main' still has a bug with our pwpolicy_force_password_change rule.  Try using 'big_sur' or 'catalina'. 

We will be merging 'big_sur' into 'main' soon, which will resolve the issue.

**Reply by erefneb** [reply]
The catalina and big sur branches work.
Good to know this is being addressed.
